typing-react-state/Exercise3/problem/App.tsx

```diff
@@ -0,0 +1,57 @@
+// Make changes to the useRefs in order to correct TS errors
+// Take note of the state's structure while adding types.
+// Bonus question - What happens if we use useState
+// instead of useRef?
```

Comment:
what if similar to slide ex useref to give focus to first input that has an error?